modules/ROOT/pages/shared-resources.adoc

```diff
@@ -12,6 +12,9 @@ Shared resources allow multiple development teams to work in parallel using the
 * Share the connection to persistent storage.
 * Share services between apps through a well-defined interface.
 * Ensure consistency between apps upon any changes because the configuration is only set in one place.
+* Share metadata, if the Mule domain project is open on the workspace while you are working on the linked Mule project.
++
+If the Mule domain project is not open on the workspace, you must input metadata manually on the linked Mule project.
```

Comment:
I feel something odd when reading the contrast between having the project open vs. not open in the workspace.
Maybe the second sentence can be added as a [NOTE] or [TIP]?
I'll approve the PR because I don't think its wrong, it's just that something looks out of place for me but cannot determine what.

Reply:
I am OK about making it a note. I haven't seen a standard way that Mulesoft doc does this. Usually it says "Note that." Salesforce doc has a specific DITA element for Note.

Reply:
Hi Cristian, we have been working with Sofia, what do you think about the following text:

"* Improve API development experience by sharing the associated domain metadata only if the domain project is open at Anypoint Studio, otherwise the metadata must be setted manually"

This one, instead of the proposal of adding line 15, 16 and 17

Reply:
The "Improve API development experience" sounds a bit too marketing-like to me. We can pull the wording out of the bullets - something like this:

To share the metadata, keep the Mule domain project open in Anypoint Studio. Otherwise, you must enter the metadata manually on the linked Mule projects. 

You might have better wording for "linked Mule projects."

Reply:
Sounds good to us.